frontend/src/Components/game.js

```diff
@@ -73,10 +68,35 @@ class Game extends React.Component {
       prevProps.location.pathname !== this.props.location.pathname &&
       this.props.history.action === 'POP'
     ) {
-      this.routeChangeHandler(this.props.location.pathname);
+      if (
```

Comment:
I can't remember why I need to do this, I thought I would just be able to redirect to this.props.location.pathname

Reply:
It appears I need it, because I'm trying to undo the state of being open.

Reply:
An alternate solution I found is to replace this code with, this.routeChangeHandler(this.props.location.pathname); and then in the default case of routeChangeHandler set the open states to false.